coding-the-legislation/25_vectorial_computing.md

```diff
@@ -20,6 +20,33 @@ array([45, 42, 17])
 
 Basic operations such as `+` or `*` behave the same way on vectors than on numbers, you can thus use them in OpenFisca. However, some operations and structures must be adapted.
 
+## Scalars
+
+As a first-time developer, it is natural to write such a formula
+```py
+def formula(person):
+    rebate = 610
+    return rebate
+```
+Unfortunately, this will generate an error similar to
+> The formula ‘rates_rebate@2016’ should return a Numpy array; instead it returned ‘610.0’ of ’<type ‘float’>’.
```

Comment:
Add a blank line above as the blockquote vs paragraph parsing can vary depending on the Markdown parser.